application/common/models/Reset.php

```diff
@@ -164,14 +164,14 @@ public function send()
          */
         $this->trackAttempt('send');
 
-        if ($this->user->do_not_disclose) {
+        if ($this->user->hide === 'yes') {
             try {
                 $this->sendAll();
             } catch (\Throwable $t) {
                 $log = [
                     'action' => 'reset send all',
                     'status' => 'error',
-                    'error' => 'Exception during password reset for undisclosed account. ' . $t->getMessage()
+                    'error' => 'Exception during password reset for account with hide flag. ' . $t->getMessage()
```

Comment:
Very minor suggestion, but this could also be phrased "for hidden account":
```suggestion
                    'error' => 'Exception during password reset for hidden account. ' . $t->getMessage()
```

Reply:
I think the benefit of consistent search text was on my mind. As such 'hidden' wouldn't be found when searching for 'hide'.

Reply:
Ahh, that makes sense. Feel free to leave it as you have it.